themes/src/main/resources/theme/keycloak.v2/login/theme.properties

```
@@ -0,0 +1,22 @@
+parent=keycloak
+import=common/keycloak
+
+styles=css/login.css
```

This loads the original login styles, which might have incompatibilities with the new stuff we're doing. Perhaps we can clear this instead?
```suggestion
styles=
```